### DISCUSSION TITLE
How do you slow the bot down?  Where do you set a custom speed in between checks?

### DISCUSSION THREAD

**Discussion by frozen-cdn** [openbullet/OpenBullet2#1140] [Q&A]
Hello, experimenting with this app and have not been able to locate a place to set the bot speed.  Even if I lower to one bot, the thing tries as fast as it can by the looks of it.  Where can you enter something like a 5-10 second delay between checks or whatever?  Maybe even advanced settings like pause after X queries?

I'm using latest OpenBullet2-web version

Thank you!

**Comment by openbullet** [comment]
Hello, you can add a delay block to your config. The value is in milliseconds. You can also limit the CPM (I think it's in the config settings).

**Comment by Spyderzz** [comment]
Add Delay Block just before the POST Request Block 
5-6K millisecs is just fine
